app/assets/javascripts/application.js

```diff
@@ -11,6 +11,38 @@
 // about supported directives.
 //
 //= require jquery
+//= require jquery.cookie
+//= require bootstrap-sprockets
 //= require jquery_ujs
-//= require turbolinks
 //= require_tree .
+
```

Comment:
It's cool that you're interested in using languages other than Ruby/Rails, but it was not necessary for this project :smiley_cat: